core/src/main/scala/org/apache/spark/rdd/RDD.scala

```diff
@@ -1296,6 +1296,7 @@ abstract class RDD[T: ClassTag](
    * an exception if called on an RDD of `Nothing` or `Null`.
    */
   def take(num: Int): Array[T] = withScope {
+    val takeRampUpRate = conf.getInt("spark.rdd.take.rampUpRate", 4)
```

Comment:
One thing is Spark SQL the other is in Core. Where should the final config live? I thought you'd want symmetry between those two functions and there doesn't seem to be config outside of sparkconf that is available in both places.